frontend/src/App.tsx

```diff
@@ -10,38 +10,41 @@ import Admin from "./pages/admin";
 import Dashboard from "./pages/dashboard";
 import Theme from "./components/Theme";
 import ThemeSwitcher from "./pages/theme";
+import Game from "./pages/game";
 
 const App = (): JSX.Element => {
-
-    return (
-        <Theme>
+  return (
+    <Theme>
+      <Body>
         <Container>
-            <BrowserRouter>
-                <Routes>
-                    <Route path="/" element={<Layout/>}>
-                        <Route path="/" element={<Navigate to="/dashboard"/>}/>
-                        <Route path="dashboard" element={<Dashboard/>}/>
-                        <Route path="leaderboard" element={<Leaderboard/>}/>
-                        <Route path="lobbies" element={<Lobbies/>}/>
-                        <Route path="login" element={<Login/>}/>
-                        <Route path="register" element={<Register/>}/>
-                        <Route path="theme" element={<ThemeSwitcher/>}/>
-                        <Route path="admin/*" element={<Admin/>}/>
-                        <Route path="*" element={<Navigate to="/"/>}/>
-                    </Route>
-                </Routes>
-            </BrowserRouter>
+          <BrowserRouter>
+            <Routes>
+              <Route path="/" element={<Navigate to="/dashboard" />} />
+              <Route path="/dashboard" element={<Dashboard />} />
```

Comment:
```suggestion
              <Route path="/dashboard" element={<ProtectedRoute>
                   <Dashboard />
              </ProtectedRoute >}
              />
```

Reply:
`<Outlet />` will be rendered inside `<ProtectedRoute />` upon succesful auth check

https://reactrouter.com/en/main/components/outlet